api/management/README.md

```diff
@@ -0,0 +1,200 @@
+# Services management
+
+The API described here are intended to assist with the operation and management
+of a Veraison services deployment. They are not meant to be utilized by the
+services' users.
+
+## Policy management
+
+Policies allow augmenting attestation scheme generated [Attestation Results] on
+per-deployment basis, by specifying additional rules to be applied to these
+results. This may result in the change of the trust vector values (and,
+correspondingly, the overall result status), and/or addition of supplementary
+claims under the `policy-added-claims` EAR extension.
+
+A policy is defined by a text document describing the rules to be applied by
+the policy engine. Currently, Veraison only support the OPA policy engine.
+Correspondingly, policies must be written in Rego language used by OPA. See
+[OPA policy README] in services repo for details.
+
+Policies are automatically retrieved, based on the attestation scheme and
+tenant id for the evidence being evaluated. Currently, only one policy is
```

Comment:
Is line 21 still relevant now ?

I am ok if you say, 

only one policy will be active per tenant/scheme combination.

As we specify:
`Retrieving all policy objects for a scheme`, this statement seems ambiguous.

Reply:
Yup, I've missed that. I've reworded that paragraph to talk about the active policies instead (also clarified that this is talking about policy usage during validation, rather this management interface).